api/types.go

```diff
@@ -59,13 +59,13 @@ type ChatRequest struct {
 type Message struct {
 	Role    string      `json:"role"` // one of ["system", "user", "assistant"]
 	Content string      `json:"content"`
-	Images  []ImageData `json:"images, omitempty"`
```

Comment:
the space here before omitempty means this was never omitted, so removing this does not change behavior from the current API

Reply:
should we return it if it's empty though?

Reply:
To match the behavior of other parts of the message I say yes, but its not very aesthetically pleasing.

Reply:
It shouldn't (for now), since this would imply the response _might_ contain an image, which it won't!

Reply:
E.g. today we get this

```
"message":{"role":"assistant","content":" right","images":null}
```

it should be

```
"message":{"role":"assistant","content":" right"}
```